content/blog/2022/11/2022-11-14-hacktoberfest-recap.adoc

```diff
@@ -0,0 +1,159 @@
+tags:
+- hacktoberfest
+- event
+- community
+- newcomer
+- outreach-programs
+author: kmartens27
+description: >
+  Hacktoberfest 2022 was a great success for the Jenkins project.
+  This post highlights Jenkins contributions in artwork, translation, documentation, security, and new features.
+links:
+  discourse: https://community.jenkins.io/t/hacktoberfest-2022/3805
+opengraph:
+  image: /images/hacktoberfest/hacktoberfest_2022_social.png
+---
+
+As November continues, we want to share some highlights from Hacktoberfest 2022!
+This year's edition was incredibly busy with lots of interesting ideas, great work, new Jenkins contributors, and returning contributors. 
+We want to thank all of the contributors and mentors for all of their hard work, as none of this would be possible without them.
+
+link:https://hacktoberfest.com/[Hacktoberfest] is an event to highlight and celebrate open-source projects like Jenkins.
+The main idea of the event is contributing to and improving open-source software.
+People are invited to submit pull requests to contribute to plugins, provide documentation, create translations, and review code among other things, to engage with open-source software. 
+Many products and services are based on open-source software, so giving back in this way continues the cycle of development.
+
+For link:https://www.jenkins.io/events/hacktoberfest/faq/[Jenkins], Hacktoberfest is an opportunity to collaborate with and expand the community, while learning and sharing with all experience levels.
+These contributions are what keeps Jenkins moving forward, so having an opportunity to celebrate both the project and the people is amazing.
+
+== Hacktoberfest results
+
+During October, the Jenkins project received over 613 pull requests from 117 different contributors.
+All of these helped to improve Jenkins, create new resources, and review the needs of Jenkins accordingly.
+From these, 531 pull requests are completed, meaning they have been merged or flagged as "hacktoberfest-approved".
+They were submitted by 95 different contributors, with 42 qualifying for link:https://hacktoberfest.com/participation/#contributors[swag] with Jenkins contributions alone!
+A total of 1,183 manually created pull requests were submitted during October in the jenkins-ci and jenkins-infra organizations.
+Overall, this appears to be a 10 to 15 percent increase in the average number of monthly submissions.
+With all of this in mind, we want to highlight some of the new and returning Jenkins contributors, including their experiences and advice for other users.
+
+== New Jenkins contributors
+
+First, we'd like to share some insight from link:https://github.com/Jagrutiti[Jagruti Tiwari]. 
+Jagruti is Based in Mumbai, India and active in various areas of open source.
+This immersion is the main component of their strategy to explore and learn new things.
+They are not afraid of challenges and have received assistance from the friendly, open-source communities. 
```

Comment:
Are you using the plural pronoun "They" at the request of the person or as a matter of style?  

If the individual has a preferred pronoun, I believe that is what we should use.

If they don't have a preferred pronoun and we know their gender, then past patterns would be to use the gender specific pronoun, rather than risking confusion for non-English native readers when they try to understand the use of the plural pronoun referring to an individual.

Reply:
Agreeing with @MarkEWaite. Unless we want to use her story and generalize it to other new contributors as a pattern. But in that case it should be made clearer.  
Based on the next sentence (inspiration), I vote for replacing "they" by "she"

Reply:
@MarkEWaite & @jmMeessen that makes total sense. I had checked profiles & other accounts to verify preferred pronouns, but did not see that information for some contributors. I tried to keep it less specific so that it would be more inclusive, but I do see the issue with translating this properly. I'll change things up and make sure that things are labeled correctly.